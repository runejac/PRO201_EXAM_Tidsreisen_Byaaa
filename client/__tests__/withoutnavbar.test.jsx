import React from "react";
import { MemoryRouter } from "react-router-dom";
import WithoutNavbar from "../src/components/WithoutNavbar.jsx";
import renderer from "react-test-renderer";

describe("WithoutNavbar", () => {
  it("shows snapshot", async () => {
    const component = renderer.create(
      <MemoryRouter>
        <WithoutNavbar />
      </MemoryRouter>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
