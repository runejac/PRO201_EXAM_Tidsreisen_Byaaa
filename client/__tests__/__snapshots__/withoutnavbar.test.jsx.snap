// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithoutNavbar shows snapshot 1`] = `null`;
